packages/contracts/src/contracts/current/protocol/Exchange/LibErrors.sol

```diff
@@ -21,13 +21,17 @@ pragma experimental ABIEncoderV2;
 
 contract LibErrors {
 
-    // Error Codes
+    // Status Codes
```

Comment:
Either call it `Errors` or `Status`, but be consistent.

Reply:
I prefer `Status`, since `SUCCESS` isn't exactly an error.
Maybe `enum OrderStatus` and rename `SUCCESS` to `ORDER_VALID`.